docs/architecture/adr-019-strict-inflation-schedule.md

```diff
@@ -55,7 +55,7 @@ Additionally, the annual provisions should be based on the total supply at the b
 |------|---------------|----------------------------|-------------------|
 | 0    | 8.00          | 100                        | 8.00              |
 | 1    | 7.20          | 108                        | 7.776             |
-| 2    | 7.20          | 115                        | 6.48              |
+| 2    | 6.48          | 115.776                    | 7.5022848         |
```

Comment:
[for reviewers]

- Inflation (%) is 6.48 based on this [line above](https://github.com/celestiaorg/celestia-app/pull/2288/files#diff-23f57de78bf2a38583ac234fde1f607911c2387e68d94acea1bfd949c747f8e2R32).
- Total supply at year start = 108 + 7.776
- Annual provisions = .0648 * 115.776